library/src/main/java/com/theguardian/bridget/thrift/$nativeConstants.java

```diff
@@ -0,0 +1,14 @@
+/**
+ * Autogenerated by Thrift Compiler (0.18.1)
+ *
+ * DO NOT EDIT UNLESS YOU ARE SURE THAT YOU KNOW WHAT YOU ARE DOING
+ *  @generated
+ */
+package com.theguardian.bridget.thrift;
+
+@SuppressWarnings({"cast", "rawtypes", "serial", "unchecked", "unused"})
+public class $nativeConstants {
+
+  public static final java.lang.String BRIDGET_VERSION = "2.0.0";
```

Comment:
Bridget version is now provided as part of generated files so it can be automated.